node/pkg/driver/messages.go

```diff
@@ -20,3 +20,5 @@ var ErrorWhileTryingToReadIQN = "Error while trying to get iqn  from string: %v.
 var ErrorUnsupportedConnectivityType = "Unsupported connectivity type : {%v}"
 
 var ErrorWhileTryingToReadFC = "Error while tring to get FC port from string: %v."
+
+var ErrorWhileTryingToGenerateNodeId = "could not fit any ports in node id: %v, length limit: %d"
```

Comment:
this error name is too generic comparing to its message. consider something like: `ErrorNoPortsCouldFitInNodeId`

Reply:
done